packer-scripts/remove-default-users

```diff
@@ -22,10 +22,7 @@ __remove_user() {
     return
   fi
 
-  ps -Fwww -u "${u}" || echo "Suppressing ps exit $?"
-  killall -TERM -w -v -u "${u}" || echo "Suppressing killall exit $?"
```

Comment:
You are correct.
The user running the packer script is `ubuntu`, so when removing the `ubuntu` user, we don't want to kill ubuntu's processes, otherwise we kill the current SSH session, which is not good.